jest.setup.js

```diff
@@ -0,0 +1,20 @@
+import '@babel/polyfill'
+
+const {
+  createProvider
+} = require('./tests/pactHelper.js')
+
+module.exports = async () => {
+  let provider = createProvider(false, false)
+  await provider.executeTest(async () => {
+  })
+  provider = createProvider(true, true)
+  await provider.executeTest(async () => {
+  })
+  provider = createProvider(true, false)
+  await provider.executeTest(async () => {
+  })
+  provider = createProvider(false, true)
+  await provider.executeTest(async () => {
+  })
+}
```

Comment:
Do we need this? there are no tests here

Reply:
This is to create empty pact files.
That way we make sure there is a file for every combination of verified/pending interactions
I've wrote a comment in the file